slybot/slybot/starturls/__init__.py

```diff
@@ -54,27 +54,31 @@ def __init__(self, spec, generators):
     def allowed_domains(self):
         if self._has_fragments:
             return self._find_fragment_domains()
-        return set([self.spec['url']])
+        return [self.spec['url']]
 
     def _find_fragment_domains(self):
         generator = self.generators[self.generator_type]
-        fragments = generator.process_fragments(self.spec)
+        fragmentLists = generator.process_fragments(self.spec)
```

Comment:
Can you change this to fragment_lists and change the other camelcase variables too?